edgelet/hsm-sys/azure-iot-hsm-c/tests/test_utils/test_utils.c

```diff
@@ -148,7 +148,7 @@ void hsm_test_util_delete_dir(const char *dir_guid)
         FALSE,
         NULL,
         NULL };
-    status = SHFileOperation(&shfo);
+    status = SHFileOperationA(&shfo);
```

Comment:
Do you need to change the type for `shfo` to `SHFILEOPSTRUCTA` as well?

Reply:
Yes that missing. Will fix.